Classes/EventStore/EventTransport.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Neos\Cqrs\EventStore;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Cqrs\Event\EventInterface;
+
+/**
+ * Wrapper for a event that was loaded from the Event Store.
+ * It contains the raw event data including version & metadata (storedEvent)
+ * and the converted EventInterface instance (event)
+ */
+final class EventTransport
```

Comment:
Good question, I also struggled with this.
So the reason is that `EventStream::current()` can only return one object. And if that was just the `storedEvent` one would have to take care of the deserialization everywhere it's used.
(So in `AbstractEventSourcedRepository::findByIdentifier()` and `ProjectionManager::replay()` and more to come in the framework, and whenever you read from the event store in 3rd party code)